src/index.js

```diff
@@ -45,7 +47,16 @@ const App = () => {
         setToDos([...todos, newTodoItem]);
 
         setTodoInputValue('');
-        setDateInputValue('');
+        setDateInputValue('');  
+    }
+
+    const validateDueDate = (typedDateInput) => {
+        setDateInputValue(typedDateInput);
+        if (typedDateInput < tomrISO) {
+            setIsDateInvalid(true);
+        } else {
+            setIsDateInvalid(false);
+        }
```

Comment:
In order to validate the typed in date, I need to intercept the setting of its state by adding this func `validateDueDate`